src/SmartComponents/BaselinesPage/CreateBaselineModal/CreateBaselineModal.js

```diff
@@ -175,12 +187,12 @@ class CreateBaselineModal extends Component {
     }
 
     renderActions() {
-        const { selectedBaselineIds, entities } = this.props;
-        const { baselineName, copyBaselineChecked, copySystemChecked } = this.state;
+        const { entities } = this.props;
+        const { baselineName, copyBaselineChecked, copySystemChecked, selectedBaseline } = this.state;
         let actions;
 
         if (baselineName === ''
-            || (copyBaselineChecked && selectedBaselineIds.length === 0)
+            || (copyBaselineChecked && selectedBaseline.length === 0)
```

Comment:
Is `selectedBaseline` now a baseline object? If so, I think this check should get updated to see if its undefined rather than checking its length.